release/e2e.py

```diff
@@ -1004,7 +1042,8 @@ def run_test_config(
     stop_event = multiprocessing.Event()
     result_queue = multiprocessing.Queue()
 
-    session_name = f"{test_name}_{int(time.time())}"
+    if not session_name:
```

Comment:
We now allow session name to be specified so that we can instantly run commands and test without waiting until sessions are newly started